index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;900&display=swap" rel="stylesheet">
+    <title>Capstone - Sports League</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="media.css">
+    <script src="menu.js" defer></script>
+    <script src="speakers.js" defer></script>
+</head>
+
+<body>
+    <header>
+        <nav aria-label="Menu" class="top-header">
+            <ul>
+                <li><a class="f-icon" href="#"></a></li>
+                <li><a class="t-icon" href="#"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </nav>
+        <nav aria-label="Menu" class="mobile-menu">
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#main">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+            </ul>
+
+        </nav>
+        <nav aria-label="Menu" class="desktop-menu">
+            <a class="logo" href="index.html"></a>
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#main">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><button type="button">Campaign</button></li>
+            </ul>
+        </nav>
+    </header>
+    <section id="banner" class="banner">
+        <div class="banner-page">
+            <i class="menu-button"></i>
+            <div>
+                <h2>"Hello Sports Enthusiast!"</h2>
+                <h1>Welcome to the Sports League<br> Conferance 2023</h1>
```

Comment:
- [ ] Good job so far 👍🏽. I will like to bring to your notice that `h2` tag cannot be used before the ` h1` tag because the order of the headings hierarchy is from `h1 - h6` for best practices. 

I will just suggest you use `<span>` tag for the first text instead of the `h2` tag that you used so that it respects the headings hierarchy this way, thereby increasing your website accessibility 👍🏽